frame/chainrelay/eth/offchain/Cargo.toml

```diff
@@ -1,57 +1,57 @@
 [package]
 name = "darwinia-eth-offchain"
 version = "0.1.0"
-authors = ["yanganto <yanganto@gmail.com>"]
+authors = ["Darwinia Network <hello@darwinia.network>"]
+description = "FRAME pallet to relay Ethereum header automactically"
 edition = "2018"
+license = "GPL-3.0"
+homepage = "https://darwinia.network/"
+repository = "https://github.com/darwinia-network/darwinia/"
 
 [dependencies]
-num-traits = { default-features = false, version = "0.2.8" }
-
-sp-std = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", tag = "v2.0.0-alpha.darwinia.1" }
-sp-runtime = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", tag = "v2.0.0-alpha.darwinia.1" }
-sp-io = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", tag = "v2.0.0-alpha.darwinia.1" }
-sp-session = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", tag = "v2.0.0-alpha.darwinia.1" }
+# crates.io
+codec = { package = "parity-scale-codec", version = "1.0.0", default-features = false, features = ["derive"] }
+hex = { version = "0.4.2", default-features = false }
+simple-json = { version = '0.1.5', default_features = false, git = 'https://github.com/jimmychu0807/simple-json' }
 
+# github.com
 frame-support = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", tag = "v2.0.0-alpha.darwinia.1" }
 frame-system = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", tag = "v2.0.0-alpha.darwinia.1" }
 
-pallet-timestamp = { default-features = false,  git = "https://github.com/darwinia-network/substrate.git", tag = "v2.0.0-alpha.darwinia.1" }
-pallet-eth-relay = { default-features = false, package = "darwinia-eth-relay", path = "../relay" }
+sp-runtime = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", tag = "v2.0.0-alpha.darwinia.1" }
+sp-std = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", tag = "v2.0.0-alpha.darwinia.1" }
```

Comment:
Need to upgrade dependency to v2.0.0-alpha.darwinia.2

Reply:
already on my local